src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/validation/PreCalculationValidationService.kt

```diff
@@ -12,6 +12,7 @@ import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.external.Sent
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.external.SentenceCalculationType
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model.external.SentenceCalculationType.Companion.from
 import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.service.ImportantDates.PCSC_COMMENCEMENT_DATE
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.service.ImportantDates.SHPO_OFFENCE_COMMENCEMENT_DATE
```

Comment:
Could you change this to be SENTENCING_ACT_2020_COMMENCEMENT or something, as it's not SHPO related anymore, SEC_91_END_DATE could then also be updated to use the same more generic date

Reply:
Yes that makes more sense in light of the changes.